DataLayer/Models/Category.cs

```diff
@@ -3,7 +3,7 @@
 using System.Linq;
 using System.Threading.Tasks;
 
-namespace MVCNewsPortal.Models
+namespace DataLayer.Models
 {
     public class Category
```

Comment:
this one can be like CategoryDto
But not required, it is contains in different namespace
if you need to use both models one of them can used like DataLayer.Models.Category